### PR TITLE
Add NLVM

### PR DESCRIPTION
**Pull request by def-**
Compiles Nim to LLVM IR

### CODE REVIEW

**Comment by waldyrious** [comment]
The data in this file is actually sourced from the **[mohd-akram/compilers](https://github.com/mohd-akram/compilers)** repository. Please submit your change to the [compilers.coffee](https://github.com/mohd-akram/compilers/blob/master/src/compilers.coffee) file there, and the update should be [imported here](https://github.com/mohd-akram/languages/commits/gh-pages/js/compilers.js) shortly afterwards.

File: js/compilers.js [hunk 1]
```diff
@@ -627,28 +627,35 @@
       source: 'Nim',
       target: 'C',
       type: 'Transpiler',
-      url: 'http://nim-lang.org/'
+      url: 'https://nim-lang.org/'
```
Comment (mohd-akram) [inline js/compilers.js:630]: Keep the URL changes in a separate PR.

File: js/compilers.js [hunk 2]
```diff
@@ -627,28 +627,35 @@
...
       target: 'Objective-C',
       type: 'Transpiler',
-      url: 'http://nim-lang.org/'
+      url: 'https://nim-lang.org/'
+    },
+    {
+      name: 'NLVM',
+      source: 'Nim',
+      target: 'LLVM IR',
+      type: 'Transpiler',
```
Comment (mohd-akram) [inline js/compilers.js:657]: Should be Intermediate.

**Review by mohd-akram — CHANGES_REQUESTED** [review]

**Comment by mohd-akram** [comment]
Please open a PR in [mohd-akram/compilers](https://github.com/mohd-akram/compilers) with the requested changes. Thank you!

**Pull request closed without merging**